tests/data/settings.json

```diff
@@ -228,6 +228,97 @@
     "version": 1
   },
 
+  "AB_T_100_W_25_25_25_25": {
```

Comment:
where did you get this test case?

Reply:
Wrote on my own . need to test some whitelisting logic where targeting variation > 1 and thus scaling weight ad bucketing follows.